tests/test_transforms.py

```diff
@@ -0,0 +1,245 @@
+from pathlib import Path
```

Comment:
Could you rename to `test_image_transforms.py`?

Reply:
Fixed https://github.com/huggingface/lerobot/pull/234/commits/1890637f27c5a917f54dedf70fc6fad9b60a6ee5